File: src/main/java/de/eugenbernwald/fretboardtrainer/model/Shift.java
package de.eugenbernwald.fretboardtrainer.model;

public enum Shift {

    FLAT(-1),
    NATURAL(0),
    SHARP(1);

    int value;

    Shift(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }

    public String getSymbol(){

        switch (this){
            case FLAT: return "♭";
            case NATURAL: return "";
            case SHARP: return "♯";
            default: return "INVALID";

        }
    }
}

File: src/main/java/de/eugenbernwald/fretboardtrainer/model/Shift.kt
package de.eugenbernwald.fretboardtrainer.model

enum class Shift(value: Int) {

    FLAT(-1),
    NATURAL(0),
    SHARP(1);

    val value: Int

    init {
        this.value = value
    }

    val symbol: String
        get() {
            return when (this) {
                FLAT -> "♭"
                NATURAL -> ""
                SHARP -> "♯"
            }
        }


}
